spec/jobs/lettings_log_import_job_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe LettingsLogImportJob do
```

Comment:
RSpec provides some nice test helpers for jobs:
https://relishapp.com/rspec/rspec-rails/docs/job-specs/job-spec